docs/en/wallet-solution.rst

```diff
@@ -13,38 +13,80 @@ such as Credential Issuers and Relying Parties.
+By supporting the mobile app, the Wallet Provider ensures the security and reliability of the entire Wallet Solution,
 as it is responsible for issuing the Wallet Attestation,
 which is a cryptographic proof about the authenticity and integrity of the Wallet Instance.
 
-Wallet Solution Requirements
+Requirements
 -----------------------------
 
-This section lists the requirements that are be met by Wallet Providers and Wallet Solutions.
+This section lists the requirements that are to be met by Wallet Providers, Wallet Solutions, and their Wallet Instances.
 
- - The Wallet Provider MUST offer a RESTful set of services for issuing the Wallet Attestations.
- - The Wallet Instance MUST periodically reestablish trust with its Wallet Provider.
- - The Wallet Instance MUST establish trust with other participants of the Wallet ecosystem, such as Credential Issers and Relying Parties.
+ - The Wallet Provider MUST expose a set of endpoints, exclusively available to its Wallet Solution instances, supporting the core functionalities of the Wallet Instances.
+ - The Wallet Instance MUST periodically reestablish trust with its Wallet Provider, obtaining a Wallet Attestation.
+ - The Wallet Instance MUST establish trust with other participants of the Wallet ecosystem, such as Credential Issers and Relying Parties, presenting a Wallet Attestation.
  - The Wallet Solutions MUST adhere to the specifications set by this document for obtaining Personal Identification (PID) and (Q)EAAs.
  - The Wallet Instance MUST be compatible and functional on both Android and iOS operating systems and available on the Play Store and App Store, respectively.
  - The Wallet Instance MUST provide a mechanism to verify the User's actual possession and full control of their personal device.
 
+Wallet Attestation Requirements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Wallet Attestation contains information regarding the security level of the device hosting the Wallet Instance.
+It primarily certifies the **authenticity**, **integrity**, **security**, **privacy**, and **trustworthiness** of a particular Wallet Instance.
+
+.. figure:: ../../images/static_view_wallet_instance_attestation.svg
+   :name: Wallet Solution Schema
+   :alt: The image illustrates the containment of Wallet Provider and Wallet Instances within the Wallet Solution, managed by the Wallet Provider.
+   :target: https://www.plantuml.com/plantuml/uml/VP8nJyCm48Lt_ugdTexOCw22OCY0GAeGOsMSerWuliY-fEg_9mrEPTAqw-VtNLxEtaJHGRh6AMs40rRlaS8AEgAB533H3-qS2Tu2zxPEWSF8TcrYv-mJzTOGNfzVnXXJ0wKCDorxydAUjMNNYMMVpug9OTrR7i22LlaesXlADPiOraToZWyBsgCsF-JhtFhyGyZJgNlbXVR1oX5R2YSoUdQYEzrQO1seLcfUeGXs_ot5_VzqYM6lQlRXMz6hsTccIbGHhGu2_hhfP1tBwHuZqdOUH6WuEmrKIeqtNonvXhq4ThY3Dc9xBNJv_rSwQeyfawhcZsTPIpKLKuFYSa_JyOPytJNk5m00
+
+
+The requirements for the Wallet Attestation are defined below:
+
+- The Wallet Attestation MUST contain a Wallet Instance public key.
+- The Wallet Attestation MUST use the signed JSON Web Token (JWT) format;
```

Comment:
```suggestion
- The Wallet Attestation MUST use the signed JSON Web Token (JWT) format;
```

what about COSE using wallet attestation with ISO18013-5?

Reply:
With a separate PR we will also introduce support for the mDoc format. For now we only support JWT. I have created an [issue](https://github.com/italia/eid-wallet-it-docs/issues/604) about it